cmd/account/get/account-claim.go

```diff
@@ -134,5 +110,61 @@ func (o *getAccountClaimOptions) run() error {
 		return err
 	}
 
-	return resourcePrinter.PrintObj(&accountClaim, o.Out)
+	for _, accountClaim := range accountClaims {
+		err = resourcePrinter.PrintObj(&accountClaim, o.Out)
+		if err != nil {
+			return nil
+		}
+	}
+
+	return nil
+}
+
+func GetAccountClaimFromAccountID(ctx context.Context, kubeCli client.Client, accountID string, namespace string) (accountClaims []awsv1alpha1.AccountClaim, err error) {
+	accounts := awsv1alpha1.AccountList{}
+	err = kubeCli.List(ctx, &accounts, &client.ListOptions{
+		Namespace: namespace,
+	})
+	if err != nil {
+		return
+	}
+
+	errors := []error{}
+	for _, a := range accounts.Items {
+		if a.Spec.AwsAccountID == accountID {
+			claim, claimerr := getAccountClaim(ctx, kubeCli, a.Name)
```

Comment:
minor: We could save a loop here when getting the account claim by using the name and namespace in the search - in the account CR object we store them in the claimLink and claimLinkNamespace fields

Reply:
agree this could be done